src/components/A11yHidden.jsx

```diff
@@ -0,0 +1,31 @@
+import styled, { css } from 'styled-components';
+
+const StA11yHidden = styled.span`
+  overflow: hidden;
+  position: ${(props) => (props.as === 'caption' ? 'static' : 'absolute')};
+  clip: rect(1px, 1px, 1px, 1px);
```

Comment:
세은님 테스트용으로 요기 1px들을 10px로 바꿔서 커밋한번 해보실 수 있나요 @senasoon 
```suggestion
  clip1: rect(10px, 10px, 10px, 10px);
```